nautobot/utilities/forms/widgets.py

```diff
@@ -160,6 +162,43 @@ def __init__(self, *args, **kwargs):
 
         self.attrs["data-multiple"] = 1
 
+    def get_context(self, name, value, attrs):
```

Comment:
Is similar logic needed in `APISelect`? I don't know that we use it anywhere in filter forms but it might still be useful in create/edit forms.

Reply:
Thanks, I think `APISelect` is the appropriate location for the logic